content/account/sso.textile

```diff
@@ -0,0 +1,97 @@
+---
+title: Single sign-on (SSO)
+meta_description: "Single sign-on enables users to authenticate with Ably using your own identity provider."
+meta_keywords: "sso, saml, strict mode, single sign-on"
+---
+
+Single sign-on (SSO) enables your users to authenticate via any SAML-compatible identity provider.
+
+<aside date-type='note'>
+<p>User provisioning and deprovisioning via SCIM or SAML JIT is not supported.</p>
+</aside>
+
+h2(#configure). Configure
+
+Single sign-on is restricted to Enterprise customers only and must be enabled on a per-account basis by "contacting Ably":https://ably.com/contact. Only "account owners":/account/users can configure SSO for an account.
+
+Any SAML-compatible identity provider can be used to enable SSO.
+
+The following instructions are examples of configuring SSO with "Okta":#okta and "Google Workspace":#google.
+
+h3(#okta). Okta
+
+To enable SSO using "Okta":https://www.okta.com/ as the identity provider:
+
+In your Ably account dashboard:
+
+1. Log in to your "account":https://ably.com/accounts/any.
+2. Select *Settings* from the account navigation dropdown.
+3. Toggle *Enable Single Sign-On* under the *Authentication Settings* section.
+4. Note down the *Single sign-on URL* and *Audience URI* values.
+
+In your Okta account:
+
+Use the "Okta guide":https://developer.okta.com/docs/guides/saml-application-setup/overview/ for enabling SSO.
+
+1. Upload the Ably logo.
+2. Select *EmailAddress* for the *Name ID format* field.
+3. Select *Email* for the *Application username* field.
+4. Ably requires users' full names. Okta can share user profile fields as SAML attributes.
+5. Ensure you assign users to the newly created Okta application.
+6. Note down the *Identity provider metadata* from Okta.
+
+In your Ably account dashboard:
+
+1. Log in to your "account":https://ably.com/accounts/any.
+2. Select *Settings* from the account navigation dropdown.
+3. Complete the SSO fields with the values obtained from Okta:
+** *Identity Provider Single Sign-On URL*
+** *Identity Provider Issuer*
+** *X.509 Certificate*
+4. *Save* the authentication settings.
+
+h3(#google). Google Workspace
+
+To enable SSO using "Google Workspace":https://workspace.google.com/ as the identity provider:
```

Comment:
It might just be me, but this and line 57 don't read right. Would it be better to have them under the same line? You're kind of telling them to do this.. then in the dashboard to this.

`To enable SSO using "Google Workspace":https://workspace.google.com/ as the identity provider, in your Ably account dashboard:` or something similar?

Reply:
There are two sections, basically, `In your Ably account dashboard` and `In your Google Workspace account`, but I see how it is a bit confusing. 

Maybe then on this line something like `To enable SSO using "Google Workspace":https://workspace.google.com/ as the identity provider, first you need to do the following in your Ably account dashboard:`, and then instead of lines 63 and 65 do something like `Then use the "Google Workspace guide":https://support.google.com/a/answer/6087519?hl=en to enable SSO in your Google Workspace account:`

Reply:
Good point: I've reworded this and the same situation in the Okta instructions.